test/e2e/run.sh

```diff
@@ -93,7 +93,7 @@ usage() {
     echo "    distro:  Linux distribution to be / already installed on vm."
     echo "             Supported values: centos-7, centos-8, debian-10, debian-sid"
     echo "                 fedora, opensuse, opensuse-tumbleweed, sles,"
-    echo "                 ubuntu-18.10, ubuntu-20.04"
+    echo "                 ubuntu-{18.04,20.04,20.10,21.04}"
```

Comment:
Hmm, shouldn't we skip 20.10 as it's been EOL'd, like 18.10?

Reply:
Thanks @marquiz, good point. Dropped 20.10.